doc/sphinx/electrostatics.rst

```diff
@@ -147,6 +148,29 @@ fluctuations in energy.
 
 For :math:`\kappa = 0`, this corresponds to the plain Coulomb potential.
 
+.. _Reaction Field method:
+
+Reaction Field method
+---------------------
+
+:class:`espressomd.electrostatics.ReactionField`
+
+The Reaction Field electrostatic potential is defined by
+
+  .. math:: U^{C-RF} = C \cdot q_1 q_2 \left[\frac{1}{r} - \frac{B r^2}{2r_{\mathrm{cut}}^3} -\frac{1 - B/2}{r_{\mathrm{cut}}}\right] \quad \mathrm{for}\quad r<r_{\mathrm{cut}}
```

Comment:
```suggestion
  .. math:: U^{C-RF} = C \cdot q_1 q_2 \left[\frac{1}{r} - \frac{B r^2}{2r_{\mathrm{cut}}^3} - \frac{1 - B/2}{r_{\mathrm{cut}}}\right] \quad \mathrm{for}\quad r<r_{\mathrm{cut}}
```

Reply:
I'll add it to my typos todo list for an upcoming PR